packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -32,14 +130,86 @@ export default class AwsS3Multipart extends BasePlugin {
     this.#client = new RequestClient(uppy, opts)
 
     const defaultOptions = {
-      timeout: 30 * 1000,
+      timeout: 30 * 1000, // todo this in no longer in use. either implement or remove.
       limit: 0,
-      retryDelays: [0, 1000, 3000, 5000],
+      retryDelays: [0, 1000, 3000, 5000], // todo this in no longer in use. either implement or remove.
```

Comment:
This was added with https://github.com/transloadit/uppy/commit/5012eb4124a2a4e490a4e68303f9a7b910e8c5f1, we should probably add it back but with support for 429 etc.